packages/upset/src/components/Header/AttributeButton.tsx

```diff
@@ -29,51 +29,42 @@ export const AttributeButton: FC<Props> = ({ label, sortable = false }) => {
   };
 
   const handleOnClick = () => {
-    if (sortable) {
-      if (sortBy !== label) {
-        sortByHeader('Ascending');
-      } else {
-        sortByHeader(sortByOrder === 'Ascending' ? 'Descending' : 'Ascending');
-      }
+    if (sortBy !== label) {
+      sortByHeader('Ascending');
+    } else {
+      sortByHeader(sortByOrder === 'Ascending' ? 'Descending' : 'Ascending');
     }
   };
 
   const handleContextMenuClose = () => {
     setContextMenu(null);
   };
 
-  const getMenuItems = () => {
-    const items = [];
-    if (sortable) {
-      items.push(
-        {
-          label: `Sort by ${label} - Ascending`,
-          onClick: () => {
-            sortByHeader('Ascending');
-            handleContextMenuClose();
-          },
-          disabled: sortBy === label && sortByOrder === 'Ascending',
-        },
-        {
-          label: `Sort by ${label} - Descending`,
-          onClick: () => {
-            sortByHeader('Descending');
-            handleContextMenuClose();
-          },
-          disabled: sortBy === label && sortByOrder === 'Descending',
-        },
-      );
-    }
-    items.push({
+  const getMenuItems = () => [
```

Comment:
Initially I wasn't sure how I felt about this notation but I think it makes sense for the situation. I do think it would be helpful to add a type for this `const` to make things a bit more comprehensible, ie `(() => (label: string, onClick: () => void, disabled: boolean)[]). Possibly create a `MenuItem` type to make this more comprehensible as well.